Sources/OLEKit/FileWrapper+Extension.swift

```diff
@@ -0,0 +1,221 @@
+import Foundation
+
+#if os(iOS) || os(watchOS) || os(tvOS) || os(macOS)
+
+extension FileWrapper: Reader {
+  func seek(toOffset: Int) {
+    seek(toOffset: toOffset)
+  }
+
+  func readData(ofLength: Int) -> Data {
+    readData(ofLength: ofLength)
+  }
+
+  func readDataToEnd() -> Data {
+    readDataToEnd()
+  }
+
+  func loadSector(_ header: Header, index: UInt32) throws -> DataReader {
```

Comment:
This code here is still duplicated with [what is declared in `extension FileHandle` in `FAT.swift`](https://github.com/CoreOffice/OLEKit/blob/59b18787edbb010152d5c0a31ecb8527b966ab4f/Sources/OLEKit/FAT.swift#L42). What we probably need is changing `extension FileHandle` to a protocol extension, that is `extension Reader`. Then you'd declare protocol conformance to `Reader` on both `FileWrapper` and `FileHandle`, which would get the `loadSector`, `loadFAT` and `oleStream` implementations through the protocol extension. What do you think about this approach?

Reply:
Cool idea! I applied it to commit c8ba566. So, there is only an extension for Reader, instead of redundant extension FileHandle and FileWrapper. Also, there is only one preprocessor in OLEFile.init which checks Apple platforms.